craft_archives/repo/package_repository.py

```diff
@@ -27,6 +27,26 @@
 
 from . import errors
 
+UCA_ARCHIVE = "http://ubuntu-cloud.archive.canonical.com/ubuntu"
+UCA_NETLOC = urlparse(UCA_ARCHIVE).netloc
+UCA_VALID_POCKETS = ["updates", "proposed"]
+UCA_DEFAULT_POCKET = UCA_VALID_POCKETS[0]
+UCA_KEYRING_PATH = "/usr/share/keyrings/ubuntu-cloud-keyring.gpg"
+
+UCA_KEYRING_PACKAGE = "ubuntu-cloud-keyring"
+RELEASE_MAP = {
+    "rocky": "bionic",
+    "stein": "bionic",
+    "train": "bionic",
+    "ussuri": "bionic",
+    "victoria": "focal",
+    "wallaby": "focal",
+    "xena": "focal",
+    "yoga": "focal",
+    "zed": "jammy",
+    "antelope": "jammy",
```

Comment:
we can forward plan this a bit and add "bobcat": "jammy" as well

Reply:
Removed the `RELEASE_MAP` dict